platform/upload/BUILD.bazel

```diff
@@ -11,6 +11,9 @@ ocis = {
     "flannel-monitor": "//flannel-monitor:oci",
     "kube-dns-main": "//kube-dns:kube-dns-main",
     "kube-dns-sidecar": "//kube-dns:kube-dns-sidecar",
+    "metallb-controller": "//metallb:controller",
+    "metallb-speaker": "//metallb:speaker",
+    "metallb-mirror": "//metallb:mirror",
```

Comment:
I see your change to fix this; I notice that `         "mirror_digest": ":mirror.ocidigest", ` means that this OCI will probably still be built, which I don't think we need, since `mirror_digest` doesn't appear to be used anywhere.

Further, do we even need the build defined for the mirror OCI anymore, if we don't need to build it?